set.mm

```diff
@@ -510554,6 +510819,1209 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+          WNWBBFWSWMWALWSWLVTVMGWSWHVPWKVSIWSWEVNWGVOGWSWDHUGWQWRUSZUTWSWFJUIWQ
+          WRVAZUTVBWSWIVQWJVRGWSWFJUGXAUTWSWDHUIWTUTVBVBVCVDVEVFVHVIVJ $.
+      $}
+
+      $d .- a b t $.  $d .0. a b t $.  $d .x. a b t $.  $d B a b t $.
+      $d B r $.  $d B s $.  $d E a b t $.  $d F a b t $.  $d G a b t $.
+      $d H a b t $.  $d R a b t $.  $d S a b t $.  $d S r $.  $d S s $.
+      $d T t $.  $d U a b t $.  $d U r $.  $d U s $.  $d V a b t $.
+      $d a ph s $.  $d b ph t $.  $d ph r $.
+      erlbrd.u $e |- ( ph -> U = <. E , G >. ) $.
+      erlbrd.v $e |- ( ph -> V = <. F , H >. ) $.
+      erlbrd.e $e |- ( ph -> E e. B ) $.
+      erlbrd.f $e |- ( ph -> F e. B ) $.
+      erlbrd.g $e |- ( ph -> G e. S ) $.
+      erlbrd.h $e |- ( ph -> H e. S ) $.
+      erlbrd.1 $e |- ( ph -> T e. S ) $.
+      erlbrd.2 $e |- ( ph -> ( T .x. ( ( E .x. H ) .- ( F .x. G ) ) ) = .0. )
+        $.
+      $( Deduce the ring localization equivalence relation.  (Contributed by
+         Thierry Arnoux, 4-May-2025.) $)
+      erlbrd $p |- ( ph -> U .~ V ) $=
+        ( vt va vb wbr cxp wcel wa cv co wceq wrex opelxpd eqeltrd simpr oveq1d
+        cop jca eqeq1d rspcedvd c1st cfv c2nd copab erlval eqtrid simprl fveq2d
+        op1stg syl2anc eqtrd adantr simprr op2ndg oveq12d oveq2d rexbidv brab2d
+        cerl eqid mpbir2and ) AHNCUOHBEUPZUQZNWLUQZURULUSZILGUTZJKGUTZMUTZGUTZP
+        VAZULEVBZAWMWNAHIKVGZWLUDAIKBEUFUHVCVDANJLVGZWLUEAJLBEUGUIVCVDVHAWTFWRG
+        UTZPVAULFEUJAWOFVAZURZWSXDPXFWOFWRGAXEVEVFVIUKVJAWOUMUSZVKVLZUNUSZVMVLZ
+        GUTZXIVKVLZXGVMVLZGUTZMUTZGUTZPVAZULEVBZXAUMUNHNCWLWLACDEWIUTXGWLUQXIWL
+        UQURXRURUMUNVNZRAULBXSDEGMOWLPUMUNQUAUBUCWLWJXSWJSTVOVPAXGHVAZXINVAZURZ
+        URZXQWTULEYCXPWSPYCXOWRWOGYCXKWPXNWQMYCXHIXJLGYCXHHVKVLZIYCXGHVKAXTYAVQ
+        ZVRAYDIVAYBAYDXBVKVLZIAHXBVKUDVRAIBUQZKEUQZYFIVAUFUHIKBEVSVTWAWBWAYCXJN
+        VMVLZLYCXINVMAXTYAWCZVRAYILVAYBAYIXCVMVLZLANXCVMUEVRAJBUQZLEUQZYKLVAUGU
+        IJLBEWDVTWAWBWAWEYCXLJXMKGYCXLNVKVLZJYCXINVKYJVRAYNJVAYBAYNXCVKVLZJANXC
+        VKUEVRAYLYMYOJVAUGUIJLBEVSVTWAWBWAYCXMHVMVLZKYCXGHVMYEVRAYPKVAYBAYPXBVM
+        VLZKAHXBVMUDVRAYGYHYQKVAUFUHIKBEWDVTWAWBWAWEWEWFVIWGWHWK $.
+    $}
+  $}
+
+  ${
+    erlbr2d.b $e |- B = ( Base ` R ) $.
+    erlbr2d.q $e |- .~ = ( R ~RL S ) $.
+    erlbr2d.r $e |- ( ph -> R e. CRing ) $.
+    erlbr2d.s $e |- ( ph -> S e. ( SubMnd ` ( mulGrp ` R ) ) ) $.
+    erlbr2d.m $e |- .x. = ( .r ` R ) $.
+    erlbr2d.u $e |- ( ph -> U = <. E , G >. ) $.
+    erlbr2d.v $e |- ( ph -> V = <. F , H >. ) $.
+    erlbr2d.e $e |- ( ph -> E e. B ) $.
+    erlbr2d.f $e |- ( ph -> F e. B ) $.
+    erlbr2d.g $e |- ( ph -> G e. S ) $.
+    erlbr2d.h $e |- ( ph -> H e. S ) $.
+    erlbr2d.1 $e |- ( ph -> T e. S ) $.
+    erlbr2d.2 $e |- ( ph -> F = ( T .x. E ) ) $.
+    erlbr2d.3 $e |- ( ph -> H = ( T .x. G ) ) $.
+    $( Deduce the ring localization equivalence relation.  (Contributed by
```

Comment:
As a suggestion:
```
Pairs ` <. E , G >. ` and ` <. T x. E , T x. G >. ` for ` T e. S ` are equivalent under the localization relation.
```

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4810/commits/f635662cfffd84458584f2c4a905b88f32b45a51